Cargo.toml

```diff
@@ -19,7 +19,7 @@ name = "dsp"
 path = "./src/lib.rs"
 
 [dependencies]
-daggy = "0.4.0"
+daggy = { git = "https://github.com/mitchmindtree/daggy", rev = "bcb36c7b", features = ["stable_dag"] }
```

Comment:
Also, we might want to update to the next proper version of `daggy` once it is released to [crates.io](https://crates.io).